### PR TITLE
Fix Z-normalization on 'y' (labels)

### PR DESCRIPTION
**Pull request by peterLingZijie**
Why:

1. Add z-normalization on `y_train` and `y_test`, as implemented in He Tong's MM
2. Add explanation in Evaluation part why z-normalization is not reversed on `y_train` and `y_test`
3. Test the code with later Python packages, including Numpy, Scipy, PyTorch and Scikit-learn

This change addresses the need by:

CBIG and external users

### CODE REVIEW

File: rs-fMRI/v1.0/meta_matching_v1.0.ipynb [hunk 1]
```diff
@@ -141,6 +141,40 @@
     "print(x_train.shape, x_test.shape, y_train.shape, y_test.shape)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we split out the subjects, we apply z-normalization to both training and testing set, using data in the training set."
```
Comment (anlijuncn) [inline rs-fMRI/v1.0/meta_matching_v1.0.ipynb]: Make it more specific. "After data split, we z-normalize the label (y) of training and test sets with training mean and std."

File: rs-fMRI/v1.1/meta_matching_v1.1.ipynb [hunk 2]
```diff
@@ -148,6 +148,40 @@
     "print(x_train.shape, x_test.shape, y_train.shape, y_test.shape)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we split out the subjects, we apply z-normalization to both training and testing set, using data in the training set."
```
Comment (anlijuncn) [inline rs-fMRI/v1.1/meta_matching_v1.1.ipynb]: Same, let the user know which variable you normalized, using which stats (mean and std).

File: rs-fMRI/v2.0/meta_matching_v2.0.ipynb [hunk 3]
```diff
@@ -133,6 +133,40 @@
     "print(x_train.shape, x_test.shape, y_train.shape, y_test.shape)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we split out the subjects, we apply z-normalization to both training and testing set, using data in the training set."
```
Comment (anlijuncn) [inline rs-fMRI/v2.0/meta_matching_v2.0.ipynb]: Same, let the user know which variable you normalized, using which stats (mean and std).

**Review by anlijuncn — CHANGES_REQUESTED** [review]
Make y normalization part more straightforward.
